deployment/articles/04-securing-liferay/05-ldap/02-configuring-ldap.markdown

```diff
@@ -25,7 +25,7 @@ a user export every time the user logs in. You can prevent updates to users'
 `lastLoginDate` fields from triggering LDAP user exports by setting the
 following property in your `portal-ext.properties` file:
 
-        users.update.last.login=false
+    users.update.last.login=false
 
```

Comment:
@mwilliams2014 Here's a property that's now not code. Did you use some automated tool to do this?

Reply:
@sez11a this is still a code block. same as above. I removed the extra space that wasn't needed